addons/account/views/report_invoice.xml

```diff
@@ -136,6 +136,10 @@
                                             <td name="td_quantity" class="text-end text-nowrap">
                                                 <span t-field="line.quantity">3.00</span>
                                                 <span t-field="line.product_uom_id"  groups="uom.group_uom">units</span>
+                                                <span t-if="line.product_uom_id != line.product_id.uom_id" groups="uom.group_uom">
+                                                    <t t-set="product_uom_qty" t-value="line.product_uom_id._compute_quantity(line.quantity, line.product_id.uom_id)"/>
+                                                    (<span t-out="product_uom_qty" t-options="{'widget': 'float', 'decimal_precision': 'Product Unit of Measure'}">3.00</span> <span t-field="line.product_id.uom_id">units</span>)
```

Comment:
Try to use `data-oe-demo` for placeholder text from now on. It will ensure that it's not printed by accident + it's less terms for the pot file and I think it doesn't even get translated anyways so it wastes the time of the translators.
```suggestion
                                                    (<span t-out="product_uom_qty" t-options="{'widget': 'float', 'decimal_precision': 'Product Unit of Measure'}" data-oe-demo="3.00"/><span t-field="line.product_id.uom_id" data-oe-demo="units"/>)
```